HISTORY.md

```diff
@@ -15,6 +15,10 @@
 * Fix the bug that WriteBatchWithIndex's SeekForPrev() doesn't see the entries with the same key.
 * Fix the bug where user comparator was sometimes fed with InternalKey instead of the user key. The bug manifests when during GenerateBottommostFiles.
 
+## 5.17.1 (10/30/2018)
+### New Features
```

Comment:
Though we may back-port your awesome feature to 5.17, it's better to add this line to the 'unreleased section'.